test/test_operations.py

```diff
@@ -1581,6 +1582,42 @@ def convert(x):
                                  convert)
     self.assertEqual(len(wids), 11)
 
+  def test_data_wrapper(self):
+
+    class PackWrapper(xu.DataWrapper):
+
+      def __init__(self, pack):
+        super(PackWrapper, self).__init__()
+        self.pack = pack
+
+      def get_tensors(self):
+        return [
+            self.pack.data, self.pack.sorted_indices, self.pack.unsorted_indices
+        ]
+
+      def from_tensors(self, tensors):
+        return nn.utils.rnn.PackedSequence(tensors[0], self.pack.batch_sizes,
+                                           tensors[1], tensors[2])
+
+    input_tensor = torch.zeros(3, 3)
```

Comment:
Is 
```
input_tensor = torch.tensor([[1, 2, 3],
                             [4, 5, 0],
                             [6, 0, 0], requires_grad=True).unsqueeze(-1)
```
a better way to setup the tensor? 
`torch.autograd.Variable` is a deprecated term after `Tensor/Variable` merge, the main difference is whether it requires grad or not ;)

Reply:
Yeah, like 1000 times better 😄